### PR TITLE
Eidaid fix

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

File: queries/public/api/entities.jq [hunk 1]
```diff
@@ -49,7 +49,8 @@ let $entities := multiplexer:entities(
 let $entities :=
   for $entity in $entities
   return {|
-    project($entity, "_id")[not $profile-name eq "japan"],
+    project($entity, "_id")[$profile-name eq "sec"],
+    { EID: $entity._id }[empty($entity.EIDs)],
```
Comment (dknochen) [inline queries/public/api/entities.jq]: are we not returning the EIDs for the japan profile?

File: queries/public/api/entities.jq [hunk 2]
```diff
@@ -49,7 +49,8 @@ let $entities := multiplexer:entities(
 let $entities :=
   for $entity in $entities
   return {|
-    project($entity, "_id")[not $profile-name eq "japan"],
+    project($entity, "_id")[$profile-name eq "sec"],
```
Comment (dknochen) [inline queries/public/api/entities.jq]: is this kept for backwards compatibility?

File: queries/public/api/filings.jq [hunk 3]
```diff
@@ -86,8 +86,8 @@ let $summaries :=
   default return
     for $archive in $summaries
     return {|
-      project($archive, "_id"),
```
Comment (dknochen) [inline queries/public/api/filings.jq]: here its not kept for backwards compatibility.

File: queries/public/api/entities.jq (hunk 2 above)
Comment (gislenius) [inline queries/public/api/entities.jq]: Yup.

File: queries/public/api/entities.jq (hunk 1 above)
Comment (gislenius) [inline queries/public/api/entities.jq]: In the Japanese profile, there is an EIDs field containing the EIDs and _id contains a UUID that should be hidden.

File: queries/public/api/filings.jq (hunk 3 above)
Comment (gislenius) [inline queries/public/api/filings.jq]: We only ensure backwards compatibility for the SEC at the moment.

File: queries/public/api/entities.jq (hunk 1 above)
Comment (dknochen) [inline queries/public/api/entities.jq]: ok, found it:
trim($entity, "_id")

**Pull request merged**